p11-kit/rpc-transport.c

```diff
@@ -642,6 +619,51 @@ rpc_transport_uninit (p11_rpc_transport *rpc)
 	p11_buffer_uninit (&rpc->options);
 }
 
+static CK_RV
+rpc_transport_authenticate (p11_rpc_client_vtable *vtable,
```

Comment:
This is named after [AUTH](https://dbus.freedesktop.org/doc/dbus-specification.html#auth-command-auth) command in D-Bus, and actually it allows the peers to exchange Unix credentials (UID and GID), so I think it's kind of authentication :-)

Reply:
In the D-Bus specification, more relevant section is [Authentication Protocol](https://dbus.freedesktop.org/doc/dbus-specification.html#auth-protocol), where not only SASL-based authentication but also capability negotiations like Unix FD passing are part of the protocol, though there is no version negotiation (I guess it's not needed at this level of the protocol, because usually the D-Bus interface version is [maintained](https://dbus.freedesktop.org/doc/dbus-api-design.html#api-versioning) at the interface level). But anyway, I don't have a strong opinion on the naming of the method.

Reply:
Me neither, feel free to keep it. I was just slightly confused.